bitnami/pinniped/Chart.yaml

```diff
@@ -27,4 +27,4 @@ maintainers:
 name: pinniped
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/pinniped
-version: 1.3.4
+version: 1.3.5
```

Comment:
Do not think we need a new version, no?

Reply:
That was to force the CI/CD pipeline to run tests, I have already removed the trigger